docs/cms-query-integration.md

```diff
@@ -37,6 +37,70 @@ add_action( 'pre_get_posts', function ( WP_Query $query ) {
 }, 20 );
 ```
 
+## Customising Searched Fields
+
+Altis will search key content fields by default, including those defined in the [field boosting config](./search-configuration/README.md#field-boosting), depending on the type of content.
+
+### Using Query Parameters
+
+`WP_Query`, `WP_Term_Query` and `WP_User_Query` support a `search_fields` parameter. This can be used override the default searched fields and even to modify how the fields are boosted using the `<field>^<boost value>` syntax.
```

Comment:
Missing word

```suggestion
`WP_Query`, `WP_Term_Query` and `WP_User_Query` support a `search_fields` parameter. This can be used to override the default searched fields and even to modify how the fields are boosted using the `<field>^<boost value>` syntax.
```

Reply:
derp